### PR TITLE
Use the upstream repo to install passenger v4

### PR DESCRIPTION
**Pull request by iuriaranda**
This way we can use the latest version of passenger

### CODE REVIEW

File: manifests/repo.pp [hunk 1]
```diff
@@ -24,23 +24,11 @@
...
-      repos       => 'main',
-      key         => '1BC1B9EF',
-      key_source  => 'http://skypackages.s3-website-eu-west-1.amazonaws.com/gpg.key',
-      include_src => false,
-    }
+  apt::source { 'passenger':
+    location    => 'https://oss-binaries.phusionpassenger.com/apt/passenger/4',
+    release     => $lsbdistcodename,
+    repos       => 'main',
+    key         => '561F9B9CAC40B2F7',
```
Comment (simonrondelez) [inline manifests/repo.pp:31]: don't think thats compatible with our apt version

**Review by venturel — APPROVED** [review]

File: manifests/repo.pp (hunk 1 above)
Comment (SamClinckspoor) [reply manifests/repo.pp:31]: yes, key must be `AC40B2F7`, because our apt module is ancient

**Review by SamClinckspoor — APPROVED** [review]
one change, otherwise 👍

File: manifests/repo.pp (hunk 1 above)
Comment (iuriaranda) [reply manifests/repo.pp:31]: https://github.com/skyscrapers/puppetlabs-apt/blob/legacy/manifests/key.pp#L60
It also accepts 16 character keys. Also tested it in vagrant and it works.

**Pull request merged**